dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/route.hibernate/Route.hbm.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd"
+        [<!ENTITY identifiableProperties SYSTEM "classpath://org/hisp/dhis/common/identifiableProperties.hbm">]>
+
+<hibernate-mapping>
+    <class name="org.hisp.dhis.route.Route" table="route">
+        <cache usage="read-write"/>
+
+        <id name="id" column="routeid">
+            <generator class="native"/>
+        </id>
+        &identifiableProperties;
+
+        <property name="name" column="name" not-null="true" unique="true" length="230"/>
+
+        <many-to-one name="createdBy" class="org.hisp.dhis.user.User" column="userid"
+                     foreign-key="fk_route_userid_userinfoid"/>
+
+        <property name="description" type="text"/>
+
+        <property name="enabled" type="boolean"/>
```

Comment:
wasn't this disabled in the class now?

Reply:
We renamed from `disabled` to `enabled`.